### ISSUE TITLE
set image height for 2.13 inch display to 128

### ISSUE THREAD

**Issue by gmoehler** [joshuatz/nfc-eink-writer#2]
I know it sounds strange, since all [documentation ](https://www.waveshare.com/wiki/Android_SDK_for_NFC-Powered_e-Paper)says that the 2.13 image has a height of 122: But in my case it did not work with that default height returning error 2 (wrong size). When i looked at the decompiled version of class `a` it reveils that the code checks against height 128 resulting in the above error. 
So i changed [the height](https://github.com/joshuatz/nfc-epaper-writer/blob/main/app/src/main/java/com/joshuatz/nfceinkwriter/Constants.kt#L22) from 122 to 128 and everything works fine. Either it is a bug in the API (probably) or waveshare has changed the dimensions of the small display (not so probable).

**Comment by joshuatz** [comment]
@gmoehler I think you are right on the money in that this is a bug in the API / SDK JAR, rather than an actual physical resolution mix-up. I honestly think it is probably just a typo from Waveshare. It doesn't surprise me either - when I built this, I actually had to reach out to Waveshare at one point because the JAR that was initially offed for download on their website was completely missing an entire class file and would not compile!

The fact that the error comes from a bounds check and not from the device itself, and that [other codebases that talk directly to the device and don't use the SDK also use `122` instead of `128`](https://github.com/RfidResearchGroup/proxmark3/blob/0d1f8ca957c0ae6f3039237889cdabe5921afe2d/client/src/cmdhfwaveshare.c#L81-L90), all point to this being an SDK issue. But rather than trying to patch the JAR myself or bug Waveshare for a fix, I'm fine with just hardcoding your workaround and adding a comment.
